### ISSUE TITLE
calcite-tree interferes with normal tab flow in Firefox

### ISSUE THREAD

**Issue by MikeTschudi** [Esri/calcite-components#3420]
### Summary

When one tabs into a calcite-tree in Firefox, tabbing out--forwards or backwards--jumps to unexpected locations.

### Actual Behavior

One can tab into a calcite-tree, but shift-tabbing back out doesn't go to previous element. Tabbing out the bottom doesn't go to the next element.

### Expected Behavior

Expect to tab into and through a calcite-tree and back again

### Reproduction Steps and Sample

https://codepen.io/miketschudi/pen/wvqpXRq

### Relevant Info

OK in Chrome 95
problem in Firefox 93

_Version_: `@esri/calcite-components@1.0.0-beta.68`

- [x] CDN
- [ ] NPM package

**Comment by jcfranco** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/calcite-5f7d10460071bf0024e66277/issues/Esri/calcite-components/3420?planning-poker) with ZenHub @caripizza

**Comment by caripizza** [comment]
@MikeTschudi are on Mac, and does this happen with other components in Firefox? It almost looks to me like this issue instead: https://css-tricks.com/proper-tabbing-to-interactive-elements-in-firefox-on-macos/

**Comment by MikeTschudi** [comment]
@caripizza, it's on Windows 10 and we've only seen tabbing into/out of issues with calcite-tree. Backwards nav used to be a problem in Chrome, but that was fixed via #683.

**Comment by y0n4** [comment]
Hi could you provide an example of an expected order of a tab key forward/backward and shift+tab sequence for this code sample? It could be an instruction by text or by video. Thanks!

**Comment by MikeTschudi** [comment]
@y0n4, I modified the https://codepen.io/miketschudi/pen/wvqpXRq to make visiting the neighbor before and after more obvious, and I switched it from *beta.68 to *beta.74.

Open the pen in Chrome or Chromium Edge:
1. Start the pen
2. Click in the demo area on "Neighbor before"
3. Tab: you enter list with "Child 1"
4. Keep tabbing past "Child 3": you land on "Neighbor after"
5. Shift-tab: you re-enter list with "Child 3"
6. Keep shift-tabbing past "Child 1": you land on "Neighbor before"
7. Repeat steps 3 through 6

Now repeat in Firefox. You'll see that step 4 doesn't match--you end up somewhere other than on "Neighbor after" (and shift-tab from that location doesn't come back to "Child 3"). If you click on "Child 1" and shift-tab, you don't end on "Neighbor before" (if you then tab, however, you'll get to "Neighbor before").
